api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -0,0 +1,179 @@
+#
```

Comment:
I would suggest segregating all Upgrade API under a dir.
Same for command would help with code structure.

`api/python/provisioner/upgrade/set_swupgrade_repo.py`
`api/python/provisioner/commands/upgrade/swupgrade.py`

Command should be a high level call and internally it should import the essential API modules.

Reply:
Also, in R2 update and Upgrade should follow same logic.
We would not want to have a separate update (may be with --update-only flag) unless absolutely necessary to have separate code path.

Reply:
This was just a suggestion, which is being incorporated by @andkononykhin2 already in one of his upcoming commits.
No change needed here.